uuid-ballerina/uuid_errors.bal

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) 2020 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+# Represents the UUID error type with the message and the cause.
+public type UuidError distinct error;
```

Comment:
Do we need this Error type. We normally don't this naming convention because the word `uuid` duplicates,
Shall we remove this and make only one type for now

Reply:
Updated in b3a0634